beeline/middleware/awslambda/test_awslambda.py

```diff
@@ -173,10 +173,17 @@ def foo(event, context):
 
             self.assertEqual(foo(None, None), 1)
 
+            @awslambda.beeline_wrapper()
+            def foo(event, context):
+                return 1
+
+            self.assertEqual(foo(None, None), 1)
```

Comment:
As @martin308 pointed out, we have a duplicate function name which the linter isn't happy with. The suggestion below should fix it.

```suggestion
            def bar(event, context):
                return 1

            self.assertEqual(bar(None, None), 1)
```